preset/kafka/preset_test.go

```diff
@@ -29,7 +29,11 @@ func TestPreset(t *testing.T) {
 	}
 
 	p := kafka.Preset(
-		kafka.WithTopics("topic-1", "topic-2"),
+		kafka.WithTopics("topic-1"),
+		kafka.WithTopicConfigs(kafka.TopicConfig{
+			Topic:         "topic-2",
+			NumPartitions: 3,
```

Comment:
Is there any way to confirm that the topic really has 3 partitions from test code? I'm not a Kafka user, sorry for the dumb question.

Reply:
There might be, I'll have a look. I am using a different kafka library for go myself